index.html

```diff
@@ -106,11 +106,19 @@ <h1>accuratelinuxgraphs.com</h1>
     </div>
     <hr>
     
+
+    <div class="content-item graph">
+      <img class="graph-img" src="files/img/distro-difficult.png.png">
+      <p class="graph-description">linux ditro easy to hurd</p>
+      <a class="graph-download-button" onclick='forceDownload("files/img/distro-difficult.png", "distro-difficult.png")'><img src="files/img/download.png"></a>
+    </div>
+    <hr>
+    
     <div class="content-item cool-fact">
       <h3 style="text-decoration: underline;">COOL LINUX FACT!</h3>
       <p>linux was created by linus torvalds in 1935</p>
       <br>
-      <img src="files/img/linux-torvald.jpg" width="500px" height="500px">
+      <a href="https://youtube.com/watch?v=dQw4w9WgXcQ" target="_blank"><img src="files/img/linux-torvald.jpg" width="500px" height="500px"></a>
```

Comment:
idea: we host a gif of never gonna give you up and the song and when you click on linus it replaces linus with the gif and plays the song

prevents anti autoplay and just closing the page while its loading

Reply:
I have no skills of js, this is the best I can do tbh

Reply:
implemented in main